x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -307,6 +307,7 @@ var sysmon = (function () {
             return;
         }
         evt.Put(field, winlogbeat.splitCommandLine(commandLine));
+        evt.Put("process.command_line", commandLine);
```

Comment:
Check out where this function is called. It's used in two different contexts so hard-coding the target field won't work. I suggest changing the `field` parameter to be the target namespace. Like this, then update the callers.

```
var splitCommandLine = function(evt, namespace) {
        var commandLine = evt.Get(field);
        if (!commandLine) {
            return;
        }
        evt.Put(namespace+".args", winlogbeat.splitCommandLine(commandLine));
        evt.Put(namespace+".command_line", commandLine);
}
```